Modules/_testcapimodule.c

```diff
@@ -3382,6 +3382,7 @@ slot_tp_del(PyObject *self)
         else
             Py_DECREF(res);
     }
+    Py_DECREF(tp_del);
```

Comment:
nit How about move this code into just after `del = _PyType_Lookup(Py_TYPE(self), tp_del);`

Reply:
done